lockfile_generator/src/lib.rs

```diff
@@ -43,7 +48,7 @@ pub trait Generator {
             .collect::<Result<Vec<_>>>()?;
 
         // Generate lockfile at the target location.
-        let mut command = self.command();
+        let mut command = self.command(&canonicalized);
```

Comment:
Since this is run from `project_path`, we could send just the filename here instead of the absolute path. It would make the commands shorter, but requires a bit more complicated code...

Reply:
I slightly prefer the current full path but would not object to the shorter command.